### ISSUE TITLE
Schedule: Droppable For External Events

### ISSUE THREAD

**Issue by cagataycivici** [primefaces/primefaces#64]
See;

http://fullcalendar.io/docs/dropping/

**Comment by BirolYildiz** [comment]
Are there any plans to implement this?  cc @melloware

**Comment by melloware** [comment]
Well its been open for 5 years so most likely the only way it will get implemented is if a community member has this need and donates a quality PR or you can sponsor it by paying PrimeTek either through PRO account or 1 time Case Quote: https://www.primefaces.org/support/

**Comment by stolp** [comment]
Just to make this clear for me, this feature request is solely about dropping external events, and not a drag-drop reordering of already existing events?

**Comment by BirolYildiz** [comment]
> Just to make this clear for me, this feature request is solely about dropping external events, and not a drag-drop reordering of already existing events?

Yes, that's my understanding. Here's a [demo](https://fullcalendar.io/docs/external-dragging-demo) from FullCalendar. Drag-drop reordering is already possible, if you look at the PF showcase.
